### ISSUE TITLE
Vue SFC support for pug

### ISSUE THREAD

**Issue by binhonglee** [vitest-dev/vitest#264]
### Describe the bug

When the template in SFC Vue files has `language="pug"` specified, it raises `[Vue warn]: Component is missing template or render function.` and expect nothing to be rendered.

### Reproduction

- Clone [vitest-ts-pug](https://github.com/binhonglee/vitest-ts-pug)
- Run `npm i && npm run test` 

### System Info

```shell
System:
    OS: Linux 4.19 Ubuntu 18.04.4 LTS (Bionic Beaver)
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 8.47 GB / 12.42 GB
    Container: Yes
    Shell: 5.4.2 - /usr/bin/zsh
  Binaries:
    Node: 16.5.0 - /tmp/fnm_multishells/277_1640150130225/bin/node
    Yarn: 1.22.5 - /usr/bin/yarn
    npm: 7.19.1 - /tmp/fnm_multishells/277_1640150130225/bin/npm
  npmPackages:
    @vitejs/plugin-vue: ^2.0.0 => 2.0.1 
    vite: ^2.7.2 => 2.7.4 
    vitest: ^0.0.103 => 0.0.103
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by AGPDev** [comment]
we have this problem too.

**Comment by sheremet-va** [comment]
Fixed in [v0.0.139](https://github.com/vitest-dev/vitest/releases/tag/v0.0.139)
